build.sh

```diff
@@ -1,5 +1,23 @@
 #! /bin/sh
 
+which git 2> /dev/null > /dev/null
+if [ $? -ne 0 ]; then
+    echo "git must be installed" > /dev/stderr
```

Comment:
And these can be shortened to `>&2 echo "..."`

Reply:
Shorter, maybe, but clearer ... ?

Reply:
Up to you. I don't feel strongly.

Reply:
Although, on reflection, `echo ... >&2` is fine, given that we use similar notation earlier.